python/ray/services.py

```diff
@@ -493,45 +499,32 @@ def start_redis(node_ip_address,
 
     if use_credis is None:
         use_credis = ("RAY_USE_NEW_GCS" in os.environ)
-    if use_credis and password is not None:
```

Comment:
Are there any behavior changes in the code for starting Redis? Or is it just moving the code around?

Reply:
I moved this code around to reduce unnecessary if-else branches.